docs/support.rst

```diff
@@ -16,7 +16,7 @@ Slack
 
 The Wagtail Slack workspace is open to all users and developers of Wagtail. To join, head to: `https://wagtail.io/slack/ <https://wagtail.io/slack/>`_
 
-Please use the **#support** channel for support questions. Support is provided by members of the Wagtail community on a voluntary basis, and we cannot guarantee that questions will be answered quickly (or at all). If you want to see this resource succeed, please consider sticking around to help out! Also, please keep in mind that many of Wagtail's core and expert developers prefer to handle support queries on a non-realtime basis through Stack Overflow, and questions asked there may well get a better response.
+Please use the **#support** channel for support questions. Support is provided by members of the Wagtail community on a voluntary basis, and we cannot guarantee that questions will be answered quickly (or at all). If you want to see this resource succeed, please consider sticking around to help out! Also, please keep in mind that many of Wagtail's core and expert developers prefer to handle support queries on a non-real time basis through Stack Overflow, and questions asked there may well get a better response.
```

Comment:
Granted "non-realtime" is probably an invented word, but "non-real time" makes it worse - it reads as "time that is not real" :-)

We should probably re-word this, but it's hard to come up with an alternative that captures the same meaning... maybe "prefer to handle support queries on their own schedule through Stack Overflow"?